app/proguard-rules.pro

```diff
@@ -32,4 +32,36 @@
 -keepattributes SourceFile,LineNumberTable
 -keep class com.hianalytics.android.**{*;}
 -keep class com.huawei.updatesdk.**{*;}
--keep class com.huawei.hms.**{*;}
\ No newline at end of file
+-keep class com.huawei.hms.**{*;}
+
+-assumenosideeffects class ly.count.android.sdk.ModuleLog {
+    public void v(...);
+    public void i(...);
+    public void w(...);
+    public void d(...);
+    public void e(...);
+}
+
+-keep class ly.count.android.demo.** { *; }
```

Comment:
this line is optional, it is not needed. I added this because we do not have much logic